docs/resources/webhook.md

```diff
@@ -13,17 +13,52 @@ Only available for dedicated subscription plans.
 
 ## Example Usage
 
+<details>
+ <summary>
+    <b>
+      <i>Enable webhook before v1.30.0</i>
+    </b>
+  </summary>
+
+Doesn't support updating the resource, makes all argument using `ForceNew` behaviour.
+Any changes to an argument will destroy and re-create the resource. Also no longer support
+for using `retry_interval` argument in the backend, even though it's set to be required.
+
```

Comment:
I wonder if we need to keep documentation for older versions in main? Maybe this is a special case as it involves `ForceNew` and the "warning" is warranted.

With that said, I think the `after v1.30.0` example should go first, then the example for the old versions.

Reply:
Yeah, would be good to keep the information for the older version. Will play around and see what can be done. One thought is to create a separate page for the older person and provide a link it from the new webhook page. This will keep the main page clean from notes and "ifs". Especially with these behaviour changes.

Reply:
I think we can keep it here instead of creating a new page. I think it is more risk we forget about that page.

We can remove the info about the older version after a while/some number of releases.

Reply:
True will also create reference in registry. Tried adding it last in the latest commit.

Reply:
Can be seen at: https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/webhook-retries/docs/resources/webhook.md#versions